Resources/Maps/_Mono/Shuttles/Nfsd/anura.yml

```
@@ -557,414 +570,414 @@
     - type: CircularShieldRadar
 - proto: AirAlarm
   entities:
-  - uid: 2
+  - uid: 3
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)